gemini/use-cases/retrieval-augmented-generation/intra_knowledge_qna.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        model="gemini-pro",
</pre><p>Change to <code>gemini-1.0-pro</code> </p><p>Could also make this a variable, since the models will probably keep updating. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/483/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>